mmv1/products/bigquery/terraform.yaml

```
@@ -109,6 +117,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         vars:
           private: "private"
           public: "public"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "bigquery_dataset_access_authorized_routine"
+        skip_test: true  # not importable
+        primary_resource_id: "access"
```

In this example, the routine is actually set on `google_bigquery_dataset_access.authorized_routine`, which is not the expected resource type, so you'll need to set that explicitly as well.

```suggestion
        primary_resource_type: "google_bigquery_dataset_access"
        primary_resource_id: "authorized_routine"
```